src/IssueViz/Editor/QuickActions/QuickFixes/QuickFixSuggestedAction.cs

```diff
@@ -64,6 +66,7 @@ public override void Invoke(CancellationToken cancellationToken)
                 textEdit.Replace(updatedSpan, edit.Edit.Text);
             }
 
+            issueViz.InvalidateSpan();
```

Comment:
Can't validate this method call in tests, because it's an extension method.

Reply:
You can test the expected behavior.
InvalidateSpan sets the span to empty, so you can create an IssueViz with a non-empty span and assert that it becomes empty afterwards. You can look at a similar behavior in `SetCurrentFilePath_FilePathIsNull_SpanIsInvalidated`